travel-advice-publisher/Makefile

```diff
@@ -0,0 +1,3 @@
+travel-advice-publisher: ../travel-advice-publisher
```

Comment:
`asset-manager content-store publishing-api` should be added to the dependencies